models/targeting/probabilistic.py

```diff
@@ -8,6 +10,13 @@
 # average time for SpyCas9 cutting in perfect match case
 average_cut_time = 60.0
 
+# distribution of indel size from CRISPResso
```

Comment:
delete the extra line, I'd like to keep the constants lumped (planning to move them later)